### DISCUSSION TITLE
Star Formation outside Galaxies

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#146] [Astrophysics and Physics]
When I was younger, I always assumed stars were constantly being formed out of the endless interstellar and intergalactic medium of dust by gravity. Currently, however, modern mainstream astrophysics indicates that stars can only form inside galaxies and galaxies are said to only have finite reserves of star-forming gas (though there are a myriad of other ways galaxies can accrue more gas over time). Science has discovered the existence of “rogue stars” that do exist outside of galaxies, but these are believed to have been stars that were ejected from their home galaxies or stars that formed out of dust that was stripped from a galaxy. What are you’re thoughts on the matter? Is star formation confined to galaxies or can it occur in intergalactic space? If star formation does only occur in galaxies, how does the universe stay out of equilibrium? On a related note, is galaxy formation itself still ongoing? Because I believe most conventional cosmologists believe galaxies and protogalaxies can only form in the early universe.

**Comment by bgreenman** [comment]
Stars might be able to form outside galaxies,if there are regions of compressed intergalatic gas clouds

**Comment by RedshiftDrift** [comment]
@MJoseph315 the model you had when you were younger seems more accurate, except that stars don't come uniquely from dust but also from lumps of gases (mainly hydrogen).  Additional conditions for the formation of stars are:
- high enough density of gas/dust, and
- a low enough temperature (otherwise thermal motion is faster than the escape velocity of the gravity generated by the lump of gas/dust).

The largest clumps of gas are found in galaxies, and that's where they are the coolest, so stars will predominantly form inside galaxies.  There is cold hydrogen in intergalactic space, but the density is so low that stars will very rarely form out there.

As for the 'rogue stars' there are many hypotheses.  If they are kicked out of the galaxy, it must be through a near-collision with another, more massive star.

**Reply by bgreenman** [reply]
Both ideas are correct, there a two types of extra-galatic stars one ejected from galaxies and the rare situations of high density gas/dust with low temperatures
